.rubocop.yml

```diff
@@ -61,6 +61,11 @@ RSpec/MultipleExpectations:
 Style/Documentation:
   Enabled: false
 
+Style/EndOfLine:
```

Comment:
Right, it was calling and EndOfLine offense on `spec/models/king_spec.rb`, so I added that file under Style/EndOfLine in `rubocop.yml` . . . which I assume was the right thing to do?